src/WignerSymbols.jl

```diff
@@ -365,13 +448,83 @@ function compute6jseries(β₁, β₂, β₃, α₁, α₂, α₃, α₄)
     return Base.unsafe_rational(totalnum, totalden)
 end
 
+# compute the sum appearing in the 9j symbol
+function compute9jseries(a, b, c, d, e, f, g, h, j)
+    I1 = max(abs(h - d), abs(b - f), abs(a - j))
+    I2 = min(h + d, b + f, a + j)
+    krange = I1:I2
+ 
+    sum(krange) do k
+        p = iseven(2k) ? big(2k + 1) : -big(2k + 1)
+
+        b₁ = let (m₁, m₂, m₃, m₄, m₅, m₆) = (a, b, c, f, j, k)     
+            α₁ = convert(BigInt, m₁ + m₅ - m₆)
+            α₂ = convert(BigInt, m₁ - m₅ + m₆)
+            α₃ = convert(BigInt, -m₁ + m₅ + m₆)
+            β₁ = convert(BigInt, m₁ + m₅ + m₆)
+            β₂ = convert(BigInt, m₂ + m₄ + m₆)
+            β₃ = convert(BigInt, m₃ + m₄ + m₅)
+            β₀ = convert(BigInt, m₁ + m₂ + m₃)
+            wei9jbracket(α₁, α₂, α₃, β₁, β₂, β₃, β₀)
+        end
+        
+        b₂ = let (m₁, m₂, m₃, m₄, m₅, m₆) = (f, d, e, h, b, k)
+            α₁ = convert(BigInt, m₁ + m₅ - m₆)
+            α₂ = convert(BigInt, m₁ - m₅ + m₆)
+            α₃ = convert(BigInt, -m₁ + m₅ + m₆)
+            β₁ = convert(BigInt, m₁ + m₅ + m₆)
+            β₂ = convert(BigInt, m₂ + m₄ + m₆)
+            β₃ = convert(BigInt, m₃ + m₄ + m₅)
+            β₀ = convert(BigInt, m₁ + m₂ + m₃)
+            wei9jbracket(α₁, α₂, α₃, β₁, β₂, β₃, β₀)
+        end
+
+        b₃ = let (m₁, m₂, m₃, m₄, m₅, m₆) = (h, j, g, a, d, k) 
+            α₁ = convert(BigInt, m₁ + m₅ - m₆)
+            α₂ = convert(BigInt, m₁ - m₅ + m₆)
+            α₃ = convert(BigInt, -m₁ + m₅ + m₆)
+            β₁ = convert(BigInt, m₁ + m₅ + m₆)
+            β₂ = convert(BigInt, m₂ + m₄ + m₆)
+            β₃ = convert(BigInt, m₃ + m₄ + m₅)
+            β₀ = convert(BigInt, m₁ + m₂ + m₃)
+            wei9jbracket(α₁, α₂, α₃, β₁, β₂, β₃, β₀)
+        end
+        
+        p * b₁ * b₂ * b₃
+    end
+end
+
+export wei9jbracket
```

Comment:
Not sure if this needs to be exported. I assume it is an internal function and users would only call `wigner9j`?

Reply:
Oops - that's a leftover from some spot-testing while I was writing my first pass. I don't think it needs exporting either. Will remove.